site/app/controllers/admin/AdminGradeableController.php

```diff
@@ -129,16 +128,24 @@ private function modifyGradeable($edit_gradeable) {
             $gradeable->setIsRepository(false);
             $gradeable->setSubdirectory("");
             $gradeable->setPointPrecision(floatval($_POST['point_precision']));
+            $is_team_assignment = (isset($_POST['team_assignment']) && $_POST['team_assignment']=='yes') ? true : false;
```

Comment:
let's change the value of the radio button on the form to be 'true'/'false' to match the other radio buttons (same with 'peer_grading'). Or change the new radio buttons to be 'yes'/'no', I don't have any particular feeling in either direction.

We can also make a private function here that is:
```
private function isRadioButtonTrue($name) {
    return isset($_POST[$name]) && $_POST[$name] === 'true';
}
```
and use that for setting all of these radio buttons instead of just repeating this identical check 6 times.